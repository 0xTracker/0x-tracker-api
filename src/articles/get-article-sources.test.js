const getArticleSources = require('./get-article-sources');

it('should get all article sources', () => {
  const sources = getArticleSources();

  expect(sources).toMatchSnapshot();
});
