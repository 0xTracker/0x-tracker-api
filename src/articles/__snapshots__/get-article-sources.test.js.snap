// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get all article sources 1`] = `
Object {
  "0xproject": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/0x.png",
    "name": "0x",
    "slug": "0x",
    "type": "other",
    "url": "https://0x.org",
  },
  "amadeus": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/amadeus-relay.png",
    "name": "Amadeus Relay",
    "slug": "amadeus-relay",
    "type": "other",
    "url": "https://amadeusrelay.org/",
  },
  "boxSwap": Object {
    "relayer": "box-swap",
    "type": "other",
  },
  "dYdX": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/dydx.png",
    "name": "dYdX",
    "slug": "dydx",
    "type": "other",
    "url": "https://dydx.exchange/",
  },
  "ddex": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/ddex.png",
    "name": "DDEX",
    "slug": "ddex",
    "type": "relayer",
    "url": "https://ddex.io",
  },
  "dharma": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/dharma.png",
    "name": "Dharma",
    "slug": "dharma",
    "type": "other",
    "url": "https://dharma.io/",
  },
  "ercdex": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/erc-dex.png",
    "name": "ERC dEX",
    "slug": "erc-dex",
    "type": "relayer",
    "url": "https://ercdex.com",
  },
  "ethfinex": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/ethfinex.png",
    "name": "Ethfinex",
    "slug": "ethfinex",
    "type": "other",
    "url": "https://www.ethfinex.com/",
  },
  "kinalpha": Object {
    "name": "Kin Alpha",
    "slug": "kin-alpha",
    "type": "other",
  },
  "ledgerDex": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/ledger-dex.png",
    "name": "LedgerDex",
    "slug": "ledger-dex",
    "type": "relayer",
    "url": "https://ledgerdex.com",
  },
  "openRelay": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/open-relay.png",
    "name": "OpenRelay",
    "slug": "open-relay",
    "type": "relayer",
    "url": "https://openrelay.xyz",
  },
  "paradex": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/paradex.png",
    "name": "Paradex",
    "slug": "paradex",
    "type": "relayer",
    "url": "https://paradex.io",
  },
  "radarrelay": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/radar-relay.png",
    "name": "Radar Relay",
    "slug": "radar-relay",
    "type": "relayer",
    "url": "https://radarrelay.com",
  },
  "sharkRelay": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/shark-relay.png",
    "name": "Shark Relay",
    "slug": "shark-relay",
    "type": "relayer",
    "url": "https://sharkrelay.com",
  },
  "theOcean": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/the-ocean.png",
    "name": "The Ocean",
    "slug": "the-ocean",
    "type": "relayer",
    "url": "https://theocean.trade/",
  },
  "tokenmom": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/tokenmom.png",
    "name": "Tokenmom",
    "slug": "tokenmom",
    "type": "relayer",
    "url": "https://tokenmom.com",
  },
  "veil": Object {
    "imageUrl": "https://0xtracker.com/assets/logos/veil.png",
    "name": "Veil",
    "slug": "veil",
    "type": "other",
    "url": "https://veil.co",
  },
}
`;
